src/trio/_channel.py

```diff
@@ -547,7 +547,7 @@ async def context_manager(
                     yield wrapped_recv_chan
                 # User has exited context manager, cancel to immediately close the
                 # abandoned generator if it's still alive.
-                nursery.cancel_scope.cancel()
+                nursery.cancel_scope.cancel(reason="user exited context manager")
```

Comment:
```suggestion
                nursery.cancel_scope.cancel(reason="exited as_safe_channel context manager")
```
Saying the user exited is either vacuous or possibly-false, but noting _which_ context manager might help.